# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/reiserfsprogs/loff_t.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- reiserfsprogs-3.6.25/include/misc.h.vanilla	2021-01-19 17:25:09.551477382 +0100
+++ reiserfsprogs-3.6.25/include/misc.h	2021-01-19 17:27:32.677484766 +0100
@@ -12,7 +12,10 @@
 #  include "config.h"
 #endif
 
+#include <stdint.h>
+
 #define _DEFAULT_SOURCE
+#define __USE_MISC
 
 #include <stdio.h>
 #include <unistd.h>
